packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/IpNetworkField.tsx

```diff
@@ -48,17 +45,16 @@ function IpNetwork({
 }: IPvAnyNetworkFieldProps) {
     const { formRowStyle } = useWithOrchestratorTheme(getCommonFormFieldStyles);
 
-    const t = useTranslations('pydanticForms');
-    const [selectedPrefix, setSelectedPrefix] = useState<IpBlock | undefined>(
-        undefined,
-    );
-    const [manualOverride, setManualOverride] = useState(false);
+    const [selectedPrefix] = useState<string | undefined>(value);
```

Comment:
If the functionality mentioned above can in fact be removed and we never use a setSeletectedPrefix anymore than there is no need to keep this value in state.